app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/edits/NoteEditsUploader.kt

```diff
@@ -26,15 +29,17 @@ class NoteEditsUploader @Inject constructor(
      *  Drops any edits where the upload failed because of a conflict but keeps any notes where
      *  the upload failed because attached photos could not be uploaded (so it can try again
      *  later). */
-    suspend fun upload() = mutex.withLock { withContext(Dispatchers.IO) {
-        // first look if any images have not been activated yet
-        uploadMissedImageActivations()
-        // then do the usual stuff
-        uploadEdits()
-    } }
+    suspend fun upload() = mutex.withLock {
+        withContext(Dispatchers.IO) {
+            // first look if any images have not been activated yet
+            uploadMissedImageActivations()
+            // then do the usual stuff
+            uploadEdits()
+        }
+    }
```

Comment:
I think this is from my running format from android studio on the whole file. Should I try to remove all these changes? or leave as is?

Reply:
Personally:

- If it were *my* PR, I probably would, because I'm comfortable mucking around with git; it's (usually) not that much effort for me to pull out changes into a separate commit, which can go in a separate PR.
- If it were my *repository*, I would not insist on you doing it; I would merge as-is. I would also suggest that next time you commit logical pieces separately (e.g. "reformat with Android Studio"), because it makes it easier to revert parts individually.
    - <details><summary>Note on git tooling</summary>
       Android Studio's git plugin makes this quite difficult, because it automatically stages every change you make. If you use the git command-line (or most other git tools), you can pick and choose which of your current changes to add to the commit. This way, if you've done a some changes both before and after reformatting, you don't need to revert it all in order to commit. I use `git add --patch` (`-p` for short) to stage just the changes I care about.
       </details>

In this case— @westnordost do you care about this reformat being mixed with the rest of the PR?